gem2rpm/smart_proxy_plugin.spec.erb

```diff
@@ -1,17 +1,18 @@
-# FIXME:
-#   1. Edit foreman-proxy requirement and specify minimum version
-#   2. Delete these lines
-#
+# Generated from <%= package.spec.file_name %> by gem2rpm -*- rpm-spec -*-
 # template: smart_proxy_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
 %global gem_name <%= spec.name %>
 %global plugin_name <%= spec.name.sub(/\Asmart_proxy_/, '') %>
 
-%global foreman_proxy_min_version FIXME
-%global foreman_proxy_dir %{_datarootdir}/foreman-proxy
+%global foreman_proxy_min_version 1.24
+%global foreman_proxy_dir /usr/share/foreman-proxy
 %global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
 %global foreman_proxy_settingsd_dir %{_sysconfdir}/foreman-proxy/settings.d
+%global smart_proxy_dynflow_bundlerd_dir %{?rhel:/opt/theforeman/tfm/root/}%{_datadir}/smart_proxy_dynflow_core/bundler.d
```

Comment:
I think the `%{_datadir}` value actually includes the SCL path automatically within the SCL env so this may end up being redundant.

Reply:
So I had a look:

```
# yum -y install https://yum.theforeman.org/releases/nightly/el7/x86_64/foreman-release.rpm && yum -y install foreman-release-scl && yum -y install tfm-build
# rpm --eval '%{?scl:%scl_package rubygem-smart_proxy_plugins}%{_datadir}'









BuildRequires: scl-utils-build
%if "tfm" == "rubygem-smart_proxy_plugins"
Requires: tfm-runtime
Provides: scl-package(tfm)
%endif

/opt/theforeman/tfm/root/usr/share
```

So I expect that this will end up as `/opt/theforeman/tfm/root//opt/theforeman/tfm/root/usr/share`